rabies/analysis_pkg/analysis_functions.py

```
@@ -242,192 +238,69 @@ def _run_interface(self, runtime):
             empty_img = sitk.GetImageFromArray(np.empty([1,1]))
             empty_file = os.path.abspath('empty.nii.gz')
             sitk.WriteImage(empty_img, empty_file)
-            NPR_extra_filename = empty_file
+            CPCA_extra_filename = empty_file
 
-        setattr(self, 'NPR_prior_timecourse_csv', NPR_prior_timecourse_csv)
-        setattr(self, 'NPR_extra_timecourse_csv', NPR_extra_timecourse_csv)
-        setattr(self, 'NPR_prior_filename', NPR_prior_filename)
-        setattr(self, 'NPR_extra_filename', NPR_extra_filename)
-        setattr(self, 'optimize_report', optimize_report_file)
+        setattr(self, 'CPCA_prior_timecourse_csv', CPCA_prior_timecourse_csv)
```

Ruff (0.11.9)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)